site/docs/v1/tech/tutorials/start-and-stop.adoc

```diff
@@ -62,33 +67,33 @@ If you'd prefer to use the native Elastic and Tomcat scripts you may follow the
 .Start FusionAuth
 ----
 /usr/local/fusionauth/fusionauth-search/elasticsearch/bin/elasticsearch
-/usr/local/fusionauth/fusionauth-app/apache-tomcat/bin/catalina.sh start
+/usr/local/fusionauth/fusionauth-app/bin/start.sh
```

Comment:
Should this page mention `startup` and `shutdown` scripts that start/stop Elastic and FusionAuth in one step?